### ISSUE TITLE
Typescript clients generated from OpenAPI spec generate methods that return void if the endpoint's content media type is */*

### ISSUE THREAD

**Issue by Norbo11** [fern-api/fern-typescript#483]
When an OpenAPI spec contains a `*/*` response type for some endpoint, the generated client function does not return the model returned by the endpoint. It returns void instead.

For example, this spec:
```yaml
  /ledger/{id}/balance:
    get:
      tags:
      - ledger-api
      operationId: getLedgerBalance
      parameters:
      - name: id
        in: path
        required: true
        schema:
          type: integer
          format: int64
      responses:
        "200":
          description: OK
          content:
            '*/*':
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/BalancePoint'
```

Generates this client method:

```typescript
    getLedgerBalance(id: number): Promise<void>;
```

Changing the `*/*` key nested under the `content` part of the response to `application/json` will generate the correct client method:

```typescript
    getLedgerBalance(id: number): Promise<FinanceApi.BalancePoint[]>;
```

`fern` should warn about such cases and maybe default to JSON so that it can attempt to do something useful if the OpenAPI spec is badly-behaved.

**Comment by Norbo11** [comment]
Dupe of https://github.com/fern-api/fern/issues/1833

**Comment by Norbo11** [comment]
Dupe of https://github.com/fern-api/fern/issues/1833
